src/features/training-program/training-program.tsx

```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
     courseName.includes('badge') ? 'aws fundamentals' : courseName,
     type,
   );
-
   const course = data as Course;
 
   const { title, content, image } = contentMap[courseName];
 
   const buttonLabel = courseName === 'react ru' ? 'Записаться' : 'Register';
 
+  const filteredContent =
+    type === 'Pre-school'
+      ? content.filter((component) => component.type.name !== 'Subtitle')
+      : content;
   return (
     <section className="training-program container">
       <div className="training-program content column-2">
         <div className="left">
           <Title text={title} hasAsterisk />
 
-          {content.map((component, index) => cloneElement(component, { key: index }))}
+          {filteredContent.map((component, index) => cloneElement(component, { key: index }))}
```

Comment:
To me index is not a property that is good in key. Do we have a unique Title?

Reply:
Content is just an array of React components, they don't even have a title, or anything else. That is what I mentioned in the description at PR - very strange and complicated logic, at least for me. 🤷‍♂️ But, if we look at it from the other side - these component never changing, once rendered, so, maybe, index as key is not a problem at all.